src/core/vdom/modules/ref.js

```diff
@@ -32,10 +32,11 @@ export function registerRef (vnode: VNodeWithData, isRemoval: ?boolean) {
     }
   } else {
     if (vnode.data.refInFor) {
-      if (Array.isArray(refs[key]) && refs[key].indexOf(ref) < 0) {
-        refs[key].push(ref)
-      } else {
+      if (!Array.isArray(refs[key])) {
         refs[key] = [ref]
+      } else if (refs[key].indexOf(ref) < 0) {
+        // $flow-disable-line
+        refs[key].push(ref)
```

Comment:
It's seems that `flow` is a little weak to check array type, even codes below couldn't pass.
```js
if (Array.isArray(refs[key])) {
  if (refs[key].indexOf(ref)) {
    refs[key].push(ref)
  }
}
```
So I added a suppress_comment option to `.flowconfig`.